app/jobs/sms_account_reset_cancel_job.rb

```diff
@@ -0,0 +1,14 @@
+class SmsAccountResetCancelJob < ApplicationJob
```

Comment:
The name for this makes me think it is resetting an SMS account. What do you think about `AccountResetCancelSmsAlertJob`

Reply:
That name sounds like we are cancelling an sms alert.  It appears the convention is SmsXXXJob.

Reply:
Or maybe one of these? Cancel makes it sound like we are canceling the Sms Alert.
- `AccountResetCancellationSmsAlertJob`
- `SmsAlertForAccountResetCancellationJob`

Reply:
Originally I had SmsAccountResetCancelledJob.  What do you think?

Reply:
The convention appears to also explain what the SMS is doing, such as sending an OTP or notifying about account reset, so then maybe `SmsAccountResetCancellationNotifierJob`?